DotNet/emitter.js

```diff
@@ -338,6 +348,19 @@
 			addCommand(dot_net`webDriver = eyes.Open(driver, ${appNm}, ${test.config.baselineName}` + rectangle + ');')
 		},
 		check(checkSettings = {}) {
+			if(checkSettings !== undefined && checkSettings.visualGridOptions)
+			{
+				if (!confVisualGridOptionCreated) {
+					addCommand(`var conf = eyes.GetConfiguration();`)
+					confVisualGridOptionCreated = true
+				}
+				var options = checkSettings.visualGridOptions
+				for (var key of Object.keys(options))
+				{
+					addCommand(`conf.SetVisualGridOptions(new VisualGridOption("${key}", ${options[key]}));`)
+				}
+				addCommand(`eyes.SetConfiguration(conf);`)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
				addCommand("eyes.SetConfiguration(conf);")
```